composer.json

```diff
@@ -42,7 +42,8 @@
         "php": ">=5.5.9",
         "illuminate/console": "5.2.* || 5.3.* || 5.4.* || 5.5.*",
         "illuminate/database": "5.2.* || 5.3.* || 5.4.* || 5.5.*",
-        "illuminate/filesystem": "5.2.* || 5.3.* || 5.4.* || 5.5.*"
+        "illuminate/filesystem": "5.2.* || 5.3.* || 5.4.* || 5.5.*",
+        "webpatser/laravel-uuid": "2.0.*"
```

Comment:
How is this better than https://github.com/ramsey/uuid ? And why is this needed?